core/resources/scene/scene.yaml

```diff
@@ -16,6 +16,22 @@ textures:
     water-texture:
         url: normals.jpg
         filtering: nearest
+
+# provide global fontfile names to be used for this stylesheet
+# these could be bundled with the tangram repo or could be loaded from system path
+# priority is given to bundled fonts
+fonts:
+    font1:
+        name: Roboto
+        weight: Regular
+    font2:
+        name: FireSans
+        weight: Medium
+    font3:
+        name: Roboto
```

Comment:
Is there any reason this information can't be inferred from text draw parameters? It seems like we could just infer the necessary fonts, lazily instantiate them, and remove the need for this top-level block entirely